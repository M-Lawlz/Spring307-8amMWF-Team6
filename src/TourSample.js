import React from 'react'

const TourSample = () => (
  <table>
    <thead>
      <tr>
        <th>Name</th>
        <th>Username</th>
        <th>Actions</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>TourName data</td>
         <td>Location data</td>
        <td>Description data</td>
        <td>Embed Url</td>
        <td>
          <button className="button muted-button">Add Tour</button>
          <button className="button muted-button">Edit</button>
          <button className="button muted-button">Delete</button>
        </td>
      </tr>
    </tbody>
  </table>
)

export default TourSample