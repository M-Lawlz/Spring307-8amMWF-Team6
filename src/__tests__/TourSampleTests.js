import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import TourSample from "../TourSample";

test("TourSample renders successfully", () => {
  const tourSample = shallow(<TourSample />);
  expect(toJson(tourSample)).toMatchSnapshot();
});
