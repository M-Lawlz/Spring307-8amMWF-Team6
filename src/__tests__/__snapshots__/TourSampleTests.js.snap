// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TourSample renders successfully 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Username
      </th>
      <th>
        Actions
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        TourName data
      </td>
      <td>
        Location data
      </td>
      <td>
        Description data
      </td>
      <td>
        Embed Url
      </td>
      <td>
        <button
          className="button muted-button"
        >
          Add Tour
        </button>
        <button
          className="button muted-button"
        >
          Edit
        </button>
        <button
          className="button muted-button"
        >
          Delete
        </button>
      </td>
    </tr>
  </tbody>
</table>
`;
